Assets/BossRoom/Scripts/Client/Game/State/ClientMainMenuState.cs

```diff
@@ -1,3 +1,13 @@
+using BossRoom.Scripts.Client.UI;
+using BossRoom.Scripts.Shared.Infrastructure;
+using BossRoom.Scripts.Shared.Net.UnityServices.Auth;
+using BossRoom.Scripts.Shared.Net.UnityServices.Lobbies;
+using ParrelSync;
```

Comment:
When trying to build, I get an error saying that the 'ParrelSync' namespace cannot be found

Reply:
should be fixed